### PR TITLE
Add doc for LogWriter

### LINKED ISSUE #133: Create LogWriter instance
Write a doc that explains in detail how LogWriter works, and show how to create a custom one.

### PR DESCRIPTION
**Pull request by findjigar**
Fixes #133

### CODE REVIEW

File: docs/LOG_WRITER.md [hunk 1]
```diff
@@ -1,54 +1,32 @@
+# LogWriter
 
+LogWriter takes care of deciding where to log the messages.
 
+### Prebuilt Loggers
 
+By default `Kermit` provides default `LogWriter` for each platform
 
+- `LogcatWriter` - Uses LogCat to send logs in Android
+- `CommonWriter` - Uses println to send logs in Kotlin
```
Comment (mrf7) [inline docs/LOG_WRITER.md:10]: nitpick, i'd keep the order that was there originally i.e. put `CommonWriter` first. Keeps a general->specific semantic and lines up with common being the default

File: docs/LOG_WRITER.md [hunk 2]
```diff
@@ -1,54 +1,32 @@
+# LogWriter
 
+LogWriter takes care of deciding where to log the messages.
 
+### Prebuilt Loggers
```
Comment (mrf7) [inline docs/LOG_WRITER.md:5]: *LogWriters

File: docs/LOG_WRITER.md [hunk 3]
```diff
@@ -1,54 +1,32 @@
...
-* `LogCatLogger` - Uses LogCat to send logs in Android
-* `NSLogLogger`  - Uses NSLog to send logs in iOS
-
-These can be created and passed into the Kermit object during initialization
+These can be created and passed into the `Logger` object during initialization
 ```kotlin
-val kermit = Kermit(LogcatLogger(),CommonLogger())
+Logger.setLogWriters(listOf(LogcatWriter(), CommonWriter())
 ```
+### Custom Logger
```
Comment (mrf7) [inline docs/LOG_WRITER.md:18]: Log Writer

**Review by mrf7 — CHANGES_REQUESTED** [review]
Some small changes

File: docs/LOG_WRITER.md (hunk 1 above)
Comment (findjigar) [reply docs/LOG_WRITER.md:9]: Done

File: docs/LOG_WRITER.md (hunk 2 above)
Comment (findjigar) [reply docs/LOG_WRITER.md]: Thanks. Done

File: docs/LOG_WRITER.md (hunk 3 above)
Comment (findjigar) [reply docs/LOG_WRITER.md]: Thanks. Done

**Pull request merged**